### PR TITLE
Update cozy-client

### PR DESCRIPTION
**Pull request by ptbrowne**
We were a bit late :sweat:

Had to remove withMutations otherwise we had warnings in our
tests and we do not allow warnings in Banks tests.

### CODE REVIEW

File: src/actions/accounts.spec.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import { removeStats } from './accounts'
 import CozyClient from 'cozy-client'
 
-jest.mock('cozy-client')
-
-const client = new CozyClient()
+const client = new CozyClient({})
```
Comment (Crash--) [inline src/actions/accounts.spec.js:4]: not createMockClient() ?

Comment (ptbrowne) [reply src/actions/accounts.spec.js:4]: No

Comment (ptbrowne) [reply src/actions/accounts.spec.js:4]: This is not the point of the PR, PR that is already making things better by removing withMutations.

Comment (Crash--) [reply src/actions/accounts.spec.js:4]: J'avais pas vu la ligne suivante...

**Pull request merged**